### PR TITLE
Significantly reduces human resistance to medborg's touching module.

### PR DESCRIPTION
**Pull request by Haevacht**
:cl:
fix: Booping is now harmful. If a borg boops you, it's rogue, blow it.
fix: Same with bopping. You are a metal monstrosity made living, you cannot control yourself carefully enough to not harm when attempting contact in such a manner.
/:cl:

Babby's first meme PR.

### CODE REVIEW

**Comment by TehZombehz** [comment]
Nu, not the boops.
Make BOPING harmful, if anything. Bopping is when you activate hug module in-hand for the more aggressive hugs, but instead boop. It is totally a thing.

**Comment by TehZombehz** [comment]
Nu not my boops. :(

**Comment by SweaterKittens** [comment]
Goof is that you?

**Comment by Grazyn** [comment]
So medborgs can perform complex surgery, dress a  wound with gauze, and other stuff but are can't control their strength when bopping?

**Comment by Haevacht** [comment]
Grazyn this is a clear meme made because Valentine exists.

**Comment by TehZombehz** [comment]
(Look at the branch name)

**Comment by GunHog** [comment]
It is supposed to be cute and/or annoying, not actually harmful. :-1: The branch name suggests that this is a grudge PR as well.

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -74,6 +74,7 @@
 									"<span class='notice'>You playfully boop [M] on the head!</span>")
 					user.do_attack_animation(M, ATTACK_EFFECT_BOOP)
 					playsound(loc, 'sound/weapons/tap.ogg', 50, 1, -1)
+					M.apply_damage(1) //Booping is now harmful.
```
Comment (ShadowDeath6) [inline code/game/objects/items/robot/robot_items.dm]: Aren't you supposed to define a type of damage, or does it default to brute without

Comment (TehZombehz) [inline code/game/objects/items/robot/robot_items.dm]: It defaults to brute without an argument for damage type, yes.

**Comment by Grazyn** [comment]
@Haevacht 
I hate Valentine as much as the next guy but I've seen enough grudge/meme PRs get merged that I know one shouldn't assume that a PR will be closed just because it's memeish

**Comment by PKPenguin321** [comment]
Pretty funny

**Comment by optimumtact** [comment]
This module is designed for borgs to hug people not harm them.

**Comment by Haevacht** [comment]
Graz I was gonna close it myself before the 24hr mark if it got that far. And yeah, defaults to brute.

Needs corporate but boopified.

**Pull request closed without merging**